src/OAuth2/Storage/JwtAccessToken.php

```diff
@@ -34,11 +34,11 @@ public function __construct(PublicKeyInterface $publicKeyStorage, AccessTokenInt
     public function getAccessToken($oauth_token)
     {
         // just decode the token, don't verify
-        if (!$tokenData = $this->encryptionUtil->decode($oauth_token, null, false)) {
+        if (!$tokenData = $this->tokenStorage->getAccessToken($oauth_token)) {
             return false;
         }
 
-        $client_id  = isset($tokenData['aud']) ? $tokenData['aud'] : null;
+        $client_id  = isset($tokenData['client_id']) ? $tokenData['client_id'] : null;
```

Comment:
Similarly, I think this should not need to change

```suggestion
        $client_id  = isset($tokenData['aud']) ? $tokenData['aud'] : null;
```

Reply:
This is because of the other change. When fetched from the tokenStorage, the client_id is called client_id, not aud. If you revert one of them, you need to revert both.

I actually think that when I did this change, I first started by getting the data from the tokenStorage to be able to find the key to pass into the decode() method. But then I saw that it was only used to fetch the client_id, and that we also have it in the tokenStorage data, so I just decided to use that instead.